examples/darkmode/src/components/ThemeManager.astro

```diff
@@ -0,0 +1,69 @@
+---
+type Props = {
+  "default-theme"?: "auto" | "dark" | "light";
```

Comment:
Is there a reason to this over `defaultTheme`? Personal preferences I guess? I mean I don't mind, I'm just more used to camel case

Reply:
I dont really mind either, I also usually do camel case but this was a leftover from our original draft